.github/workflows/release.yml

```diff
@@ -0,0 +1,31 @@
+name: release
+on:
+  push:
+    tags:
+      - '[0-9]*'
+jobs:
+  cockpituous:
+    runs-on: ubuntu-latest
+    container:
+      image: docker.io/cockpit/release
+    steps:
+      - name: Set up configuration and secrets
+        run: |
+          # override GitHub's bind mount from host, we don't want anything from there and it interferes with ssh
+          export HOME=$(getent passwd $(id -u) | cut -f6 -d:)
+
+          echo '${{ secrets.SSH_KNOWN_HOSTS }}' > ~/.ssh/known_hosts
+          echo '${{ secrets.FEDPKG_SSH_PUBLIC }}' > ~/.ssh/id_rsa.pub
+          echo '${{ secrets.FEDPKG_SSH_PRIVATE }}' > ~/.ssh/id_rsa
+          chmod 600 ~/.ssh/id_rsa
+          echo 'cockpit' > ~/.config/bodhi-user
+          echo '${{ secrets.GITHUB_TOKEN }}' > ~/.config/github-token
```

Comment:
what are these 'secrets'? Where does it come from?

Reply:
See the [comment above](https://github.com/cockpit-project/cockpit-ostree/pull/153#issuecomment-706545726). You should be able to see them. This still needs documenting in the README of course, I'll send a PR once everything works.

Reply:
I added a README stanza to explain how the release is triggered and where the secrets live.